wallet/tests/sc_sign_and_verify_command_test.go

```diff
@@ -96,7 +96,7 @@ func TestSignCommandWithTaintedKey(t *testing.T) {
 	})
 
 	// then
-	require.EqualError(t, err, "couldn't sign transaction: public key is tainted")
+	require.EqualError(t, err, "couldn't sign transaction: the public key is tainted")
```

Comment:
IMO it is good all log levels messages not to include short forms of any negatives etc.
Better if we do it `could not` everywhere, instead of `couldn't`. I think full form is also better for writing.
I could be a pain in the ass here, sorry if that is the case, but I just think that way.

Reply:
Alright. I think it's an understandable comment. Is it okay if we make wider changes to this in a dedicated PR ?

Because this has deeper impact than just this code (on system test especially).

It's worth spreading this change everywhere.

Reply:
@AnExsomnis I created a dedicated issue for that. Feel free to add anything missing:

- https://github.com/vegaprotocol/vega/issues/6010

Reply:
All right